Todo_app/services/todoAppBackend/mongoDB/Connect.js

```diff
@@ -0,0 +1,11 @@
+const mongooseConnection = async () => {
+  const mongoose = require("mongoose");
+  const MONGODB_URL =
+    process.env.NODE_ENV === "test"
```

Comment:
why dont u use only MONGODB_URL as env variable, and set it to the test url when u run tests?